Makefile

```diff
@@ -207,6 +207,7 @@ completion: builddir
 clean:
 	-find . -name "*.~[0-9]~" -exec rm -f {} \;
 	-find . -name "*.py[co]" -exec rm -f {} \;
+	-find . -name "__pycache__" -exec rm -rf {} \;
```

Comment:
If I remember correctly, Python 3 puts the `.py[co]` files in `__pycache__`, so I think the previous line is not needed anymore? (Same with `plugins/Makefile`.)